packages/pax_utils.rb

```diff
@@ -3,38 +3,61 @@
-      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.2.5_armv7l/pax_utils-1.2.5-chromeos-armv7l.tar.xz',
-        i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.2.5_i686/pax_utils-1.2.5-chromeos-i686.tar.xz',
-      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.2.5_x86_64/pax_utils-1.2.5-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.3.4_armv7l/pax_utils-1.3.4-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.3.4_armv7l/pax_utils-1.3.4-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.3.4_i686/pax_utils-1.3.4-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/pax_utils/1.3.4_x86_64/pax_utils-1.3.4-chromeos-x86_64.tar.zst'
   })
-  binary_sha256 ({
-     aarch64: 'edc856ff6cc451358da1fb46a2a1023594f3d243cb1c30e5aa67c9ee9ed4dac6',
-      armv7l: 'edc856ff6cc451358da1fb46a2a1023594f3d243cb1c30e5aa67c9ee9ed4dac6',
-        i686: '9254843237e4bba47da7c4ec63ca7109e32ad7fee5822a1862fdddc789c280bb',
-      x86_64: '836d8627d96ed61608cb7bee423db753fc00c67600c62386e1df98d4529c35c9',
+  binary_sha256({
+    aarch64: '54b27eaf90d7d9f0ec2852986c5549939ddba12b5232313996330ee0d9f4bdda',
+     armv7l: '54b27eaf90d7d9f0ec2852986c5549939ddba12b5232313996330ee0d9f4bdda',
+       i686: '05b1cd3d421d12e40ae555cb0b4422e332386d3ff5a6e0ee874ec9dfc94392eb',
+     x86_64: '427a8ce76e866049a418d8f727de81a9a76e3cb30f5021ef5b079ed9a3c4cacb'
   })
-
+  # The following two are only needed for build with autogen.sh
+  # depends_on 'gnulib_git' => :build
+  # depends_on 'xmlto' => :build
+  depends_on 'glibc' # R
   depends_on 'py3_pyelftools'
   depends_on 'libcap' => :build
   depends_on 'libseccomp' => :build
 
+  def self.patch
+    # https://lists.gnu.org/archive/html/bug-gnulib/2022-04/msg00075.html
+    @gnulib_patch = <<~'PATCHEOF'
+      diff --git a/gnulib/import/string.in.h b/gnulib/import/string.in.h
+      index b6840fa9121..cb344ed0498 100644
+      --- a/autotools/gnulib/string.in.h
+      +++ b/autotools/gnulib/string.in.h
+      @@ -594,6 +594,7 @@ _GL_CXXALIAS_SYS (strndup, char *, (char const *__s, size_t __n));
+       _GL_CXXALIASWARN (strndup);
+       #else
+       # if __GNUC__ >= 11
+      +#undef strndup
+       /* For -Wmismatched-dealloc: Associate strndup with free or rpl_free.  */
+       _GL_FUNCDECL_SYS (strndup, char *,
+                         (char const *__s, size_t __n)
+    PATCHEOF
+    File.write('gnulib.patch', @gnulib_patch)
+    system 'patch -Np1 -i gnulib.patch'
+  end
+
   def self.build
+    # system './autogen.sh'
     system "sed -i 's|/usr/bin/env python|/usr/bin/env python3|g' lddtree.py"
```

Comment:
Move to patch section?

Reply:
This doesn't look like it was moved?

Reply:
I don't understand why it isn't showing up. See the relevant commit on the branch for this PR: https://github.com/satmandu/chromebrew/commit/7896acf06011040c15c7722e0cbca8ad2f3bf02e

Reply:
I think you moved the commented line instead of the sed line.

Reply:
Got it. Made that change.